app/helpers/ukims_helper.rb

```diff
@@ -3,24 +3,79 @@ module UkimsHelper
   CUT_OFF_DATE = Date.new(2023, 9, 30)
 
   def uk_internal_market_scheme
```

Comment:
What do you think of a global renaming of this to `market_scheme_type`

Reply:
I think given we're going to delete the old one in a year or so we can ignore this

Reply:
Until another scheme comes out 😂 , i think this could still be worth it if im making changes elsewhere